MusicNotationKit/MusicNotationKit/NoteCollection.swift

```diff
@@ -25,6 +25,10 @@ public protocol NoteCollection {
      */
     var noteTimingCount: Int { get }
 
+    /// The grouping order defined for this `NoteCollection`
```

Comment:
Can we get a more descriptive comment? Probably make it multi line. Probably also give example of it being 3 for a triplet and always 1 for a single note.